transaction/src/manifest/generator.rs

```diff
@@ -1124,6 +1146,10 @@ fn generate_value(
                 value: ScryptoCustomValue::NonFungibleId(v),
             })
         }
+        ast::Value::Bytes(value) =>  
+            Ok( SborValue::String {
+                value: generate_string(value)?
```

Comment:
This is incorrect. Should be mapped to `SborValue::Array<u8>`

Reply:
Changed.